problems/service-placement/support/evaluation.py

```diff
@@ -0,0 +1,131 @@
+"""
+SPDX-FileCopyrightText: 2024 Noé Godinho <noe@dei.uc.pt>
+
+SPDX-License-Identifier: CC-BY-4.0

```

Comment:
I would suggest using a software license (e.g. Apache 2.0). CC-BY-4.0 was not designed for software and the creative commons people recommend against using it for software https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software

(But your choice of course)